### PR TITLE
Proofreading Sam Phippen captions (episode 8) : from 01:15 to the end of the video

### PR DESCRIPTION
**Pull request by nicoder**

### CODE REVIEW

**Comment by BonitaJane** [comment]
Thanks Nicolas

## 

  Bonnie Roworth
  bonnie@peertopeer.io

On Sun, Sep 13, 2015, at 09:40 AM, Nicolas Dermine wrote:

> You can view, comment on, or merge this pull request online at:
> 
> https://github.com/fromPeerToPeer/captions/pull/65
> 
> Commit Summary
> - change line wrap before changes
> - remove extra whitespace
> - make it closer to the original
> - fill in the inaud placeholder
> - write class, module names, Ruby code as they appear in the video
> - wrong words
> - fix typos
> - missing word
> - s/re-factor/refactor
>   File Changes
> - _M_ 8/sam-phippen-captions.srt[1](237)
>   Patch Links:
> - https://github.com/fromPeerToPeer/captions/pull/65.patch
> - https://github.com/fromPeerToPeer/captions/pull/65.diff
> 
> —
> Reply to this email directly or view it on GitHub[2]. 

Links:
1. https://github.com/fromPeerToPeer/captions/pull/65/files#diff-0
2. https://github.com/fromPeerToPeer/captions/pull/65

File: 8/sam-phippen-captions.srt [hunk 1]
```diff
@@ -4191,16 +4198,17 @@ quickly here so we get the next failure.
...
 
 919
 01:24:33,390 --> 01:24:37,424
-We'll do an attr.reader...
+We'll do an attr_reader...
 
 920
 01:24:38,810 --> 01:24:45,184
-...and then receive messages shovel method name.
+...and then received_messages << method_name.
```
Comment (nelstrom) [inline 8/sam-phippen-captions.srt:4211]: :+1:

File: 8/sam-phippen-captions.srt [hunk 2]
```diff
@@ -4678,25 +4691,27 @@ well understood Ruby protocols.
 
 1024
 01:34:31,790 --> 01:34:38,320
-2I, 2F, 2S and so on will probably get
+to_i, to_f, to_s and so on will probably get
```
Comment (nelstrom) [inline 8/sam-phippen-captions.srt:4694]: :+1:

**Comment by nelstrom** [comment]
Great job @nicoder, thanks!

**Pull request merged**